.github/workflows/build-and-test.yaml

```diff
@@ -256,6 +257,17 @@ jobs:
           cat /tmp/pip3-install.log
         fi
 
+        pkg_dir=dist
+        mkdir -p ${pkg_dir}
+        echo "downloading bdba package"
+        bdba_version=$(sed -n "/componentName: ocm.software\/ocm-gear\/bdba-client/{n;s/^[[:space:]]*version:[[:space:]]*//p;}" .ocm/component-references.yaml)
```

Comment:
it is also okay to read this from component-references; I think reading from base-component-descriptor would be slightly better, as this would not rely on this implementation-detail, but I think this is good enough / also a pragmatic choice.